### ISSUE TITLE
Manually backing up the titles of videos in playlists

### ISSUE THREAD

**Issue by el4ctr0n** [Jopik1/filmot-title-restorer#5]
Dear developer,
Can you please make it possible to manually backup the titles of the videos in my playlists to filmot.com?
Thank you for considering it.

**Comment by Jopik1** [comment]
yeah, I guess, I just need to figure an easy way to accept video ids for crawling.
